.gitignore

```diff
@@ -1,3 +1,4 @@
 node_modules/*
 dist/*
 .env
+.DS_Store
```

Comment:
is this necessary? we've not needed it before

Reply:
@vsai because every time I try to check if there's any type error in the terminal using `yarn tsc` these temp files get generated. That's why I think we should include it.